storage/src/tests/distributor/bucketdbupdatertest.cpp

```diff
@@ -580,12 +610,19 @@ class BucketDBUpdaterTest : public CppUnit::TestFixture,
 
 CPPUNIT_TEST_SUITE_REGISTRATION(BucketDBUpdaterTest);
 
+BucketDBUpdaterTest::BucketDBUpdaterTest()
+    : CppUnit::TestFixture(),
+    DistributorTestUtil(),
+    _bucketSpaces()
+{
+}
+
 void
 BucketDBUpdaterTest::testNormalUsage()
 {
     setSystemState(lib::ClusterState("distributor:2 .0.s:i .1.s:i storage:3"));
 
-    CPPUNIT_ASSERT_EQUAL(size_t(3), _sender.commands.size());
+    CPPUNIT_ASSERT_EQUAL(3 * _bucketSpaces.size(), _sender.commands.size());
```

Comment:
The pattern `n * _bucketSpaces.size()` is such a frequent occurrence that I think it would be beneficial to wrap it in a named function for the sake of understandability. Maybe something like `bucket_spaces_messages(n)`, `all_bucket_spaces(n)` or `bucket_space_adjusted(n)`?